src/app/components/mnoe-api/app-instances.svc.coffee

```diff
@@ -25,14 +25,18 @@ angular.module 'mnoEnterpriseAngular'
       # Workaround as the API is not standard (return a hash map not an array)
       # (Prefix operation by '/' to avoid data extraction)
       # TODO: Standard API
-      MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances').get().then(
-        (response) ->
-          # Save the response in the local storage
-          MnoLocalStorage.setObject(MnoeCurrentUser.user.id + "_" + LOCALSTORAGE.appInstancesKey, response.app_instances)
-          # Process the response
-          processAppInstances(response.app_instances)
+      MnoeOrganizations.getCurrentId().then(
```

Comment:
Why do you call `MnoeOrganizations.getCurrentId()` here?

It is supposed to already be called in the layout controller.